templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
```

Comment:
It seems you are using the `crispy_forms_filters` but there is no usage of it in this template. If you are not using it, it's better to remove it to keep your code clean.